### PR TITLE
feat: Configure nix access token to prevent GitHub rate limiting

### PR DESCRIPTION
**Pull request by jimmidyson**
Based on the suggestion at https://github.com/jetpack-io/devbox/pull/728/files#diff-e6fe83bc13b20d4fd513659c4b2cb8c34617a4b1f04e95b390517c42e905867fR134.

I have used this as a step in my own workflows and it works but would be great if it could be included in this action.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ inputs:
     default: ''
   sha256-checksum: # the expected SHA256 checksum of the devbox binary.
     description: 'Specify an explicit checksum for the devbox binary. For extra security on top of the existing checks in the devbox launch script'
+  configure-nix-access-token: # 'true' or 'false'
```
Comment (LucilleH) [inline action.yml:22]: Can we do `disable-nix-access-token` and default it to `false`? (Assuming that most users would like it to be enabled

**Review by LucilleH — COMMENTED** [review]
Thank you for the contribution! Would you mind updating the README to also include this param?

File: action.yml (hunk 1 above)
Comment (jimmidyson) [reply action.yml]: Done, thanks for the review :)

**Review by LucilleH — APPROVED** [review]

**Comment by jimmidyson** [comment]
Updated README, shouldn't need to rerun checks. Thanks again for the review.

**Pull request merged**